pipeline/outputs/s3.md

```diff
@@ -25,6 +25,9 @@ Records are stored in files in S3 as newline delimited JSON.
 | role\_arn | ARN of an IAM role to assume \(ex. for cross account access\). | None |
 | endpoint | Custom endpoint for the S3 API. | None |
 | sts\_endpoint | Custom endpoint for the STS API. | None |
+| canned\_acl | Predefined Canned ACL policy for S3 objects. | None |
```

Comment:
For this one, lets add a link to the canned ACL docs

Reply:
sure, have added the link for canned ACL.